src/transformState.js

```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
```

Comment:
please add the `default` case for `switch/case` construction. you can `throw Error` in that case

Reply:
))misunderstood the previous comment))